fixture_magic/management/commands/merge_fixtures.py

```diff
@@ -29,16 +21,26 @@ def handle(self, *files, **options):
         Load a bunch of json files.  Store the pk/model in a seen dictionary.
         Add all the unseen objects into output.
         """
+        seen = set()
         output = []
-        seen = {}
 
-        for f in files:
-            f = open(f)
-            data = json.loads(f.read())
+        for file_ in files:
+            with open(file_, 'r') as fp:
+                data = json.load(fp)
             for obj in data:
-                key = '%s|%s' % (obj['model'], obj['pk'])
+                key = '{}'.format(obj['model'])
```

Comment:
it's never our final key, so let's rename to avoid quite confusing code later (`key = '{}|{}'.format(key, key_unique)`):

```suggestion
                model_name = '{}'.format(obj['model'])
```